core-libs/setup/package.json

```diff
@@ -19,14 +19,14 @@
     "tslib": "^2.6.2"
   },
   "peerDependencies": {
-    "@angular/core": "^16.2.10",
+    "@angular/core": "^17.0.5",
     "@spartacus/cart": "6.6.0-1",
     "@spartacus/core": "6.6.0-1",
     "@spartacus/order": "6.6.0-1",
     "@spartacus/user": "6.6.0-1"
   },
   "optionalDependencies": {
-    "@angular/platform-server": "^16.2.10",
+    "@angular/platform-server": "^17.0.5",
     "@nguniversal/express-engine": "^16.2.0",
```

Comment:
[QUESTION]
Shall we remove it?

Reply:
Planned to remove this dependency when out `server.ts` is adjusted to the new API, so in the separate PR.